_posts/2024-01-12-release-0_4_1.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Kroxylicious release 0.4.1"
+date:   2023-01-12 00:00:00 +0000
+author: "Sam Barker"
+author_url: "https://www.github.com/SamBarker"
+categories:  releases kroxylcious topic-encryption
```

Comment:
```suggestion
categories:  releases kroxylicious topic-encryption
```

Reply:
maybe should fixup `2023-12-19-release-0_4_0.md` too

Reply:
As I had copy and pasted the `front matter` I've fixed this in https://github.com/kroxylicious/kroxylicious.github.io/pull/27/commits/b785f79e000fd2e3881bdd65cb1b4640f2296633